Makefile

```diff
@@ -44,6 +44,33 @@ ifeq ($(TEST_TYPE),coverage)
 		fi; \
 	done
 endif
+ifeq ($(TEST_TYPE),fuzzit)
+	export GO111MODULE=off
+	go get -u github.com/dvyukov/go-fuzz/go-fuzz-build
+	go get -u ./...
+	LIBFUZZER=YES make -f Makefile.fuzz cache chaos file rewrite whoami
+	wget -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.8/fuzzit_Linux_x86_64
+	chmod a+x fuzzit
+	./fuzzit auth $(FUZZIT_API_KEY)
+	./fuzzit create job --branch $(TRAVIS_BRANCH) --revision $(TRAVIS_COMMIT) cache ./cache
```

Comment:
adding each new fuzz target here, can be be cumbersome, can this be automatically be figured out?

Reply:
Agree. I think we can just add another target for fuzzit in the Makefile where we have all targets and logic anyway? sounds reasonable?

Reply:
Also, because go-fuzz doesnt support GOMODULES do you know what would be the best way to compile it in Travis and get all the dependencies?

I tried `go get -u` but it didn't work

https://travis-ci.org/coredns/coredns/jobs/568993981